src/test/java/task40/helpes/BaseTest.java

```diff
@@ -0,0 +1,43 @@
+package task40.helpes;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BaseTest {
+    protected WebDriver driver;
+    private String url;
+
+    public BaseTest(String url) {
+        this.url = url;
+    }
+
+    @BeforeEach
+    protected void setUp() {
+        driver = new ChromeDriver();
+        driver.get(this.url);
+        driver.manage().window().fullscreen();
+    }
+
+    @AfterEach
+    protected void cleanUp() {
+        driver.quit();
+    }
+
+    protected List<Integer> getUniqueList(int size)
```

Comment:
This method is not common for all test. It uses only by MultiSelectTest . So it shouldn't be there.
The previous decision was good(where you had SeleniumEasyMultiSelectPage class)
Please return this class. Also you can make the class name shorter by removing SeleniumEasy from the class name

Reply:
Done